config/settings/development.yml

```diff
@@ -1 +1 @@
-dataset_target_path : "/Users/alexostrovsky/Downloads"
\ No newline at end of file
+dataset_target_path : "/home/martin/RubymineProjects/"
```

Comment:
toto si tam nemal davat. resp. asi to aj tak bude musiet stano upravit ale nevadi